src/ert/config/field.py

```diff
@@ -115,20 +120,15 @@ def metadata(self) -> list[ParameterMetadata]:
                 key=self.name,
                 transformation=self.output_transformation,
                 dimensionality=3,
-                userdata={
-                    "data_origin": "FIELD",
-                    "nx": self.nx,
-                    "ny": self.ny,
-                    "nz": self.nz,
-                },
+                userdata={"data_origin": "FIELD", "ertbox_params": self.ertbox_params},
             )
         ]
 
     @classmethod
     def from_config_list(
         cls,
         grid_file_path: str,
-        dims: Shape,
+        ertbox_params: ErtboxParameters,
```

Comment:
I think the ertbox_params attribute should be created from within from_config_list

Reply:
I'm referencing ertbox from other methods like metadata, len etc., but perhaps I could use a cached property or something like that?